### PR TITLE
fixing some dashboard components

### PR DESCRIPTION
**Pull request by rartner**
* Replacing carousel by card
* Custom CSS classes fix
* Changing icon for "start trail"

### CODE REVIEW

File: app/javascript/packs/dashboard/components/Dashboard.vue [hunk 1]
```diff
@@ -1,13 +1,7 @@
...
-    <v-container style="max-width: 100%">
-      <v-carousel style="height: 250px">
-        <v-carousel-item
-          v-for="(item,i) in items"
-          :key="i"
-          :src="item.src"
-        ></v-carousel-item>
-      </v-carousel>
+    <v-container fluid>
+      <trail isActive isFavorite title="Trilha de introdução"></trail>
```
Comment (cuzik) [inline app/javascript/packs/dashboard/components/Dashboard.vue]: Here we don't need to pass the value `true` to `isActive` props?

File: app/javascript/packs/components/Trail.vue [hunk 2]
```diff
@@ -1,40 +1,59 @@
...
-        </v-container>
-      </v-card-media>
-      <v-card-actions>
-        <v-btn icon>
-          <v-icon>favorite</v-icon>
+  <v-card>
+    <v-card-media
+      class="white--text"
+      height="200px"
+      src="https://cdn.vuetifyjs.com/images/cards/docks.jpg"
```
Comment (cuzik) [inline app/javascript/packs/components/Trail.vue]: WDYT?

We can pass the `src` by props and keep by default `"https://cdn.vuetifyjs.com/images/cards/docks.jpg"`

File: app/views/dashboard/index.html.erb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 <%= javascript_pack_tag "dashboard/index" %>
+<%= stylesheet_pack_tag "dashboard/index" %>
```
Comment (cuzik) [inline app/views/dashboard/index.html.erb]: Nice >_<

File: app/javascript/packs/dashboard/components/Dashboard.vue (hunk 1 above)
Comment (rartner) [reply app/javascript/packs/dashboard/components/Dashboard.vue]: Not necessarily. In Vue, boolean props can be used as flags. 🔝

File: app/views/dashboard/index.html.erb (hunk 3 above)
Comment (rartner) [reply app/views/dashboard/index.html.erb]: :wine_glass:

**Pull request merged**